metis-enrichment/metis-enrichment-rest/src/main/java/eu/europeana/enrichment/rest/config/Application.java

```diff
@@ -55,6 +55,8 @@
   private int enrichmentMongoPort;
   @Value("${enrichment.mongo.database}")
   private String enrichmentMongoDatabase;
+  @Value("${enrichment.mongo.application.name}")
+  private String mongoApplicationName;
```

Comment:
How about aligning the name of the field too?:)

Reply:
Oops, will do that :)